### DISCUSSION TITLE
error 500 when trying to access wordpress via https

### DISCUSSION THREAD

**Discussion by superflay123** [evertramos/nginx-proxy-automation#354] [General]
Hello, congratulations on the good work. My problem with nginx proxy is when I try to access a wordpress site through https it always returns me error 500 (internal server error). Everything works on port 80. when I logged in to the docker image of the nginx proxy (docker exec -it proxy-web-auto / bin / sh) I see the following in the configuration file

###############
server {
server_name dobrich-bg.com;
listen 80;
access_log /var/log/nginx/access.log vhost;
include /etc/nginx/vhost.d/default;
location / {
proxy_pass http://dobrich-bg.com;
}
}
server {
server_name dobrich-bg.com;
listen 443 ssl http2;
access_log /var/log/nginx/access.log vhost;
return 500;
ssl_certificate /etc/nginx/certs/default.crt;
ssl_certificate_key /etc/nginx/certs/default.key;
###################

Where is the problem and where am I wrong, please help me
P.S.
the server is on a local network
192.168.1.222 Debian OS

**Comment by evertramos** [comment]
You need a public ip address in order to have the certificate issued... if you do check the logs for the letsencrypt certificate and send it to us without the personal information:

```bash
docker logs letsencrypt-auto
```

**Reply by superflay123** [reply]
**docker logs letsencrypt-auto**
Info: running acme-companion version v2.2.1
Generating a RSA private key
.................................++++
............................................................................................................................................................................................................................................................................................................................................................................................................................................++++
writing new private key to '/etc/nginx/certs/default.key.new'
-----
Reloading nginx docker-gen (using separate container docker-gen-auto)...
Reloading nginx (using separate container proxy-web-auto)...
{"message":"Cannot kill container: proxy-web-auto: Container 2a9dd67d4950bad6c9c671cd3424f00d86db0987a16bed0653c2e9e2d3602b2c is not running"}
Info: a default key and certificate have been created at /etc/nginx/certs/default.key and /etc/nginx/certs/default.crt.
Info: Setting up 4096 bits RFC7919 Diffie-Hellman group...
Reloading nginx docker-gen (using separate container docker-gen-auto)...
Reloading nginx (using separate container proxy-web-auto)...
{"message":"Cannot kill container: proxy-web-auto: Container 2a9dd67d4950bad6c9c671cd3424f00d86db0987a16bed0653c2e9e2d3602b2c is not running"}
2022/06/21 14:20:34 Generated '/app/letsencrypt_service_data' from 2 containers
2022/06/21 14:20:34 Running '/app/signal_le_service'
2022/06/21 14:20:34 Watching docker events
2022/06/21 14:20:34 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:20:34 Error: nginx-proxy container proxy-web-auto isn't running.
Sleep for 3600s
2022/06/21 14:22:53 Received event start for container 2a9dd67d4950
2022/06/21 14:22:58 Debounce minTimer fired
2022/06/21 14:22:58 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:34:44 Received event start for container 437a3c234c18
2022/06/21 14:34:44 Received event die for container 437a3c234c18
2022/06/21 14:34:46 Received event start for container 494a4c4059e9
2022/06/21 14:34:46 Received event start for container 437a3c234c18
2022/06/21 14:34:46 Received event die for container 437a3c234c18
2022/06/21 14:34:48 Received event start for container 437a3c234c18
2022/06/21 14:34:48 Received event die for container 437a3c234c18
2022/06/21 14:34:49 Received event start for container 437a3c234c18
2022/06/21 14:34:50 Received event die for container 437a3c234c18
2022/06/21 14:34:51 Received event start for container 437a3c234c18
2022/06/21 14:34:52 Received event die for container 437a3c234c18
2022/06/21 14:34:54 Received event start for container 437a3c234c18
2022/06/21 14:34:54 Received event die for container 437a3c234c18
2022/06/21 14:34:59 Debounce minTimer fired
2022/06/21 14:35:01 Received event start for container 437a3c234c18
2022/06/21 14:35:01 Generated '/app/letsencrypt_service_data' from 5 containers
2022/06/21 14:35:01 Running '/app/signal_le_service'
2022/06/21 14:35:01 Received event die for container 437a3c234c18
[Tue Jun 21 14:35:03 UTC 2022] Create account key ok.
[Tue Jun 21 14:35:03 UTC 2022] Registering account: https://acme-staging-v02.api.letsencrypt.org/directory
[Tue Jun 21 14:35:04 UTC 2022] Registered
[Tue Jun 21 14:35:04 UTC 2022] ACCOUNT_THUMBPRINT='iVwSYxnQ1Q'
Reloading nginx docker-gen (using separate container docker-gen-auto)...
Reloading nginx (using separate container 2a9dd67d4950bad6c9c671cd3424f00d86db0987a16bed0653c2e9e2d3602b2c)...
Creating/renewal repairpc.info certificates... (repairpc.info www.repairpc.info)
[Tue Jun 21 14:35:06 UTC 2022] Using CA: https://acme-staging-v02.api.letsencrypt.org/directory
[Tue Jun 21 14:35:06 UTC 2022] Creating domain key
2022/06/21 14:35:06 Debounce minTimer fired
2022/06/21 14:35:06 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
[Tue Jun 21 14:35:08 UTC 2022] The domain key is here: /etc/acme.sh/staging/repairpc.info/repairpc.info.key
[Tue Jun 21 14:35:08 UTC 2022] Multi domain='DNS:repairpc.info,DNS:www.repairpc.info'
[Tue Jun 21 14:35:08 UTC 2022] Getting domain auth token for each domain
2022/06/21 14:35:09 Received event start for container 437a3c234c18
2022/06/21 14:35:09 Received event die for container 437a3c234c18
[Tue Jun 21 14:35:12 UTC 2022] Getting webroot for domain='repairpc.info'
[Tue Jun 21 14:35:12 UTC 2022] Getting webroot for domain='www.repairpc.info'
[Tue Jun 21 14:35:12 UTC 2022] Verifying: repairpc.info
2022/06/21 14:35:14 Debounce minTimer fired
2022/06/21 14:35:14 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
[Tue Jun 21 14:35:16 UTC 2022] Pending
[Tue Jun 21 14:35:19 UTC 2022] Pending
[Tue Jun 21 14:35:21 UTC 2022] Pending
2022/06/21 14:35:22 Received event start for container 437a3c234c18
2022/06/21 14:35:23 Received event die for container 437a3c234c18
[Tue Jun 21 14:35:24 UTC 2022] Success
[Tue Jun 21 14:35:24 UTC 2022] Verifying: www.repairpc.info
[Tue Jun 21 14:35:28 UTC 2022] Pending
2022/06/21 14:35:28 Debounce minTimer fired
2022/06/21 14:35:28 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
[Tue Jun 21 14:35:30 UTC 2022] Pending
[Tue Jun 21 14:35:33 UTC 2022] Pending
[Tue Jun 21 14:35:36 UTC 2022] Success
[Tue Jun 21 14:35:36 UTC 2022] Verify finished, start to sign.
[Tue Jun 21 14:35:36 UTC 2022] Lets finalize the order.
[Tue Jun 21 14:35:36 UTC 2022] Le_OrderFinalize='https://acme-staging-v02.api.letsencrypt.org/acme/finalize/58183054/2909453924'
[Tue Jun 21 14:35:37 UTC 2022] Downloading cert.
[Tue Jun 21 14:35:37 UTC 2022] Le_LinkCert='https://acme-staging-v02.api.letsencrypt.org/acme/cert/fa82a2bd0026f681fc8e0d79263e90103a80'
[Tue Jun 21 14:35:38 UTC 2022] Cert success.
-----BEGIN CERTIFICATE-----

O4Gh61Q52taJVY2RkTIDHGhkmnebhiVXXIMTyTfhB8ha1wrvqk++JzPIIbLh70Ie
n1rDPZ+JSJT6B+cweac3Fmkpn6SfE1MpB+RchRN1gZplBCsARJoEO2caZMxqmHPH
EFkG0Pua2Xq4W0roR1VfxdktnalXjhpZLqLQ1RExf05hzOuzSS15vkaJMY6dP8Uo
zieZW0lEqx0=
-----END CERTIFICATE-----
[Tue Jun 21 14:35:38 UTC 2022] Your cert is in  /etc/acme.sh/staging/repairpc.info/repairpc.info.cer 
[Tue Jun 21 14:35:38 UTC 2022] Your cert key is in  /etc/acme.sh/staging/repairpc.info/repairpc.info.key 
[Tue Jun 21 14:35:38 UTC 2022] The intermediate CA cert is in  /etc/acme.sh/staging/repairpc.info/ca.cer 
[Tue Jun 21 14:35:38 UTC 2022] And the full chain certs is there:  /etc/acme.sh/staging/repairpc.info/fullchain.cer 
[Tue Jun 21 14:35:38 UTC 2022] Installing cert to:/etc/nginx/certs/_test_repairpc.info/cert.pem
[Tue Jun 21 14:35:38 UTC 2022] Installing CA to:/etc/nginx/certs/_test_repairpc.info/chain.pem
[Tue Jun 21 14:35:38 UTC 2022] Installing key to:/etc/nginx/certs/_test_repairpc.info/key.pem
[Tue Jun 21 14:35:38 UTC 2022] Installing full chain to:/etc/nginx/certs/_test_repairpc.info/fullchain.pem
Reloading nginx docker-gen (using separate container docker-gen-auto)...
Reloading nginx (using separate container 2b2c)...
Sleep for 3600s
2022/06/21 14:35:49 Received event start for container 437a3c234c18
2022/06/21 14:35:49 Received event die for container 437a3c234c18
2022/06/21 14:35:54 Debounce minTimer fired
2022/06/21 14:35:54 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:36:42 Received event start for container 437a3c234c18
2022/06/21 14:36:42 Received event die for container 437a3c234c18
2022/06/21 14:36:47 Debounce minTimer fired
2022/06/21 14:36:47 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:37:43 Received event start for container 437a3c234c18
2022/06/21 14:37:43 Received event die for container 437a3c234c18
2022/06/21 14:37:48 Debounce minTimer fired
2022/06/21 14:37:48 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:38:44 Received event start for container 437a3c234c18
2022/06/21 14:38:44 Received event die for container 437a3c234c18
2022/06/21 14:38:49 Debounce minTimer fired
2022/06/21 14:38:49 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:39:45 Received event start for container 437a3c234c18
2022/06/21 14:39:45 Received event die for container 437a3c234c18
2022/06/21 14:39:50 Debounce minTimer fired
2022/06/21 14:39:50 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:40:46 Received event start for container 437a3c234c18
2022/06/21 14:40:46 Received event die for container 437a3c234c18
2022/06/21 14:40:51 Debounce minTimer fired
2022/06/21 14:40:51 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:41:48 Received event start for container 437a3c234c18
2022/06/21 14:41:48 Received event die for container 437a3c234c18
2022/06/21 14:41:53 Debounce minTimer fired
2022/06/21 14:41:53 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:42:49 Received event start for container 437a3c234c18
2022/06/21 14:42:49 Received event die for container 437a3c234c18
2022/06/21 14:42:54 Debounce minTimer fired
2022/06/21 14:42:54 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:43:50 Received event start for container 437a3c234c18
2022/06/21 14:43:50 Received event die for container 437a3c234c18
2022/06/21 14:43:55 Debounce minTimer fired
2022/06/21 14:43:55 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:44:32 Received event die for container 494a4c4059e9
2022/06/21 14:44:33 Received event stop for container 494a4c4059e9
2022/06/21 14:44:38 Debounce minTimer fired
2022/06/21 14:44:38 Generated '/app/letsencrypt_service_data' from 4 containers
2022/06/21 14:44:38 Running '/app/signal_le_service'
Reloading nginx docker-gen (using separate container docker-gen-auto)...
Reloading nginx (using separate container 2a9dd67d4950bad6c9c671cd3424f00d86db0987a16bed0653c2e9e2d3602b2c)...
Sleep for 3600s
2022/06/21 14:44:51 Received event start for container 437a3c234c18
2022/06/21 14:44:51 Received event die for container 437a3c234c18
2022/06/21 14:44:56 Debounce minTimer fired
2022/06/21 14:44:56 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:45:52 Received event start for container 437a3c234c18
2022/06/21 14:45:52 Received event die for container 437a3c234c18
2022/06/21 14:45:57 Debounce minTimer fired
2022/06/21 14:45:57 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:46:53 Received event start for container 437a3c234c18
2022/06/21 14:46:53 Received event die for container 437a3c234c18
2022/06/21 14:46:58 Debounce minTimer fired
2022/06/21 14:46:58 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:47:42 Received event stop for container 437a3c234c18
2022/06/21 14:47:47 Debounce minTimer fired
2022/06/21 14:47:47 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:48:29 Received event start for container 437a3c234c18
2022/06/21 14:48:29 Received event die for container 437a3c234c18
2022/06/21 14:48:30 Received event start for container 494a4c4059e9
2022/06/21 14:48:30 Received event start for container 437a3c234c18
2022/06/21 14:48:31 Received event die for container 437a3c234c18
2022/06/21 14:48:32 Received event start for container 437a3c234c18
2022/06/21 14:48:32 Received event die for container 437a3c234c18
2022/06/21 14:48:34 Received event start for container 437a3c234c18
2022/06/21 14:48:34 Received event die for container 437a3c234c18
2022/06/21 14:48:36 Received event start for container 437a3c234c18
2022/06/21 14:48:36 Received event die for container 437a3c234c18
2022/06/21 14:48:38 Received event start for container 437a3c234c18
2022/06/21 14:48:38 Received event die for container 437a3c234c18
2022/06/21 14:48:43 Received event start for container 437a3c234c18
2022/06/21 14:48:43 Received event die for container 437a3c234c18
2022/06/21 14:48:48 Debounce minTimer fired
2022/06/21 14:48:48 Generated '/app/letsencrypt_service_data' from 5 containers
2022/06/21 14:48:48 Running '/app/signal_le_service'
Creating/renewal repairpc.info certificates... (repairpc.info www.repairpc.info)
[Tue Jun 21 14:48:49 UTC 2022] Domains not changed.
[Tue Jun 21 14:48:49 UTC 2022] Skip, Next renewal time is: Sat Aug 20 14:35:38 UTC 2022
[Tue Jun 21 14:48:49 UTC 2022] Add '--force' to force to renew.
Reloading nginx docker-gen (using separate container docker-gen-auto)...
Reloading nginx (using separate container 2a9dd67d4950bad6c9c671cd3424f00d86db0987a16bed0653c2e9e2d3602b2c)...
Sleep for 3600s
2022/06/21 14:48:50 Received event start for container 437a3c234c18
2022/06/21 14:48:50 Received event die for container 437a3c234c18
2022/06/21 14:48:55 Debounce minTimer fired
2022/06/21 14:48:55 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:49:04 Received event start for container 437a3c234c18
2022/06/21 14:49:05 Received event die for container 437a3c234c18
2022/06/21 14:49:10 Debounce minTimer fired
2022/06/21 14:49:10 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
2022/06/21 14:49:31 Received event start for container 437a3c234c18
2022/06/21 14:49:31 Received event die for container 437a3c234c18

**Comment by superflay123** [comment]
Is there an option in which to use your automatic nginx proxy but the certificates for a site to be taken from a file locally. My goal is to make a non-product server for local development. Can you help me with that?

**Reply by evertramos** [reply]
The certificate was issued, your https should be working, probably there is another issue in your web container... try the test script in the bin folder for the ssl with this domain and you will see it's working.
